README.md

```diff
@@ -40,29 +40,38 @@ There are several services and a couple of models used by them available.
  
  `npm test`
  
-#### Build the library:
- 
- `npm run build`
- 
-#### Try it out locally. 
- 
- We found that `npm link` doesn't fully work. You have to reference the library via `file:`. But you still need to create the link.
- 
- - Start by running:
- 
-   `npm link dist/`
- 
- - Change this:
- 
-   `"ngx-widgets": "X.X.X"`
-   
- - to this:
- 
-   `"ngx-widgets": "file:///[LOCATION-TO-NODE-MODULES]/.nvm/versions/node/v6.9.1/lib/node_modules/ngx-widgets"`
- 
- - then in the app that will use the lib:
- 
-   `npm install ngx-widgets`
+## Library Build
+
+### Production
+
+To build the Planner as a npm library, use:
+
+----
+npm run build
+----
+
+Whilst the standalone build uses webpack the library build uses gulp.
+
+The created library is located in `dist`. You shouldn't ever publish the
+build manually, instead you should let the CD pipeline do a semantic release.
+
+### Development
+
+To build  fabric8-planner as an npm library and embed it into a webapp such as
+fabric8-ui, you should:
+
+1. Run `npm run watch:library` in this directory. This will build fabric8-planner as
+a library and then set up a watch task to rebuild any ts, html and scss files you
+change.
+2. In the webapp into which you are embedding, run `npm link <path to fabric8-planner>/dist-watch`.
+This will create a symlink from `node_modules/fabric8-planner` to the `dist-watch` directory
+and install that symlinked node module into your webapp.
```

Comment:
Should we be linking planner or ngx-widgets here? I typically run 'npm link <path to ngx-widgets>/dist/'